src/components/ItemList.svelte

```diff
@@ -37,31 +37,83 @@
 </script>
 
 <style>
+  $tablet-size: 769px;
+
   .item-browser {
     max-height: 400px;
     overflow: scroll;
-    margin: $spacing-md auto $spacing-md auto;
     a {
       text-decoration: none;
     }
   }
+
+  table {
+    table-layout: fixed;
+    background: $color-light-gray-05;
+    border-collapse: collapse;
+    margin: auto;
+    width: 100%;
+    .table-header td {
+      font-weight: bold;
+    }
+    td {
+      padding: $spacing-xs;
+      border-right: 1px dotted $color-light-gray-30;
+      &:first-child {
+        width: 76%;
+        @media (max-width: $tablet-size) {
```

Comment:
Curious what the reason for these media queries is? FWIW I am not too concerned with what this site looks like on mobile since I expect 99% of its traffic to be on desktop

Reply:
When window size is reduced to about ~800px which is the case if you have 2 browser windows open side by side, long metric type names such as "memory_distribution" or "labeled_counter" would overflow:
<img width="185" alt="Screen Shot 2021-01-14 at 12 18 51 PM" src="https://user-images.githubusercontent.com/28797553/104644566-c2ce0f00-5662-11eb-813a-b1e1e26fc388.png">
I find that quite annoying so I set up the media queries to prevent it. I can remove it though if you don't find it necessary, it was more of a subjective decision as I tend to have 2 windows in split view quite often.

Reply:
Makes sense. Maybe just add a comment to let people know your rationale?